layouts/shortcodes/sitemap.html

```diff
@@ -0,0 +1,25 @@
+{{ $envURL := ( index .Site.Params.envURL (os.Getenv "BRANCH") ) }}
+{{ $envURL := cond (eq $envURL nil) .Site.Params.envURL.fed $envURL }}
+
+<div class="sitemap">
+  <h2>Main Navigation</h2>
```

Comment:
Replace "Main Navigation" with a variable and add it to the front matter in the md translations.